File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// ChuckNorrisJokes SDK exists test

require_once __DIR__ . '/../chucknorrisjokes_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = ChuckNorrisJokesSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: lua/test/exists_test.lua
-- ChuckNorrisJokes SDK exists test

local sdk = require("chuck-norris-jokes_sdk")

describe("ChuckNorrisJokesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
